### DISCUSSION TITLE
build err！SyntaxError: The requested module 'vue' does not provide an export named 'ssrContextKey'

### DISCUSSION THREAD

**Discussion by yijiiaaa** [vuepress/core#1333] [Q&A] [answered]
![image](https://github.com/vuepress/vuepress-next/assets/32707070/1496a00f-e150-443c-a0cb-d2859843fd7a)

**Comment by yijiiaaa** [comment]
"vuepress": "2.0.0-beta.62"

**Comment by kaluojushi** [comment, marked as answer]
Upgrading the version of node may help you, as I fixed this bug by upgrading from 14 to 18

**Reply by yijiiaaa** [reply]
Thanks, this method has helped me!
